pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -307,10 +307,11 @@ spec:
       openAPIV3Schema:
         type: boolean
         default: false
-  - name: TKR_KUBERNETES_SPEC
+  - name: tkrData
     schema:
       openAPIV3Schema:
         type: object
+        properties: {}
```

Comment:
@imikushin what's the recommendation in terms of definition of the TKR_DATA, to keep as generic object? for now?

Reply:
For now, yes. We could also add this to openAPIV3Schema (bc it's basically a map with arbitrary keys):
```yaml
x-kubernetes-preserve-unknown-fields: true
```

The schema is basically defined here: https://github.com/vmware-tanzu/tanzu-framework/blob/main/pkg/v2/tkr/webhook/cluster/tkr-resolver/cluster/tkrdata.go

If folks have time to translate this to openAPIV3Schema, that would be great.

Reply:
@tenczar assuming we are not doing this in this PR, can you file an issue to track possibly defining a more specific schema for this variable.